### PR TITLE
Minifed Readme

### PR DESCRIPTION
**Pull request by graysuit**
Read_me.md is intructions paper Not newspaper/story_book :)
Some long things remained untouched But my time came over

Edited Disclaimer
Converted long list to minmal tables
Converted long list to clopsable
Little minify Found a Bug
Removed a dead video link

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,31 +26,65 @@
   TO BE USED FOR EDUCATIONAL PURPOSES ONLY
 </p>
 
-The use of the HiddenEye & its resources/phishing-pages is **COMPLETE RESPONSIBILITY of the END-USER**. Developers assume NO liability and are NOT responsible for any misuse or damage caused by this program. Also we inform you that some of your your actions may be **ILLEGAL** and you **CAN NOT** use this software to test person or company without **WRITTEN PERMISSION** from them.
+The illegal and misuse of the HiddenEye & its resources/phishing-pages is **COMPLETE RESPONSIBILITY of the END-USER**. Developers assume NO liability and are NOT responsible in user crime. Using HiddenEye to test/attack any person/company without **WRITTEN PERMISSION** is **ILLEGAL**.
```
Comment (sTiKyt) [inline README.md:29]: This statement means "If you use it legally and in a right way - we take responsibility."
You can fix formatting or spelling, but you shouldn't change meaning of statements that could be used to sue me or any other DarkSec developer.

**Comment by graysuit** [comment]
Okay for disclaimer,
But you should use tables for making lists minimal,
Else it will get long n long as more peoples will contribute and more features will be added.
Should not be a story book,

And also I found a youtube dead link,
Must Fix it.

**Comment by sTiKyt** [comment]
We shouldn't use tables, instead we have to create contributors.md  and move all credit there.

**Comment by graysuit** [comment]
Yeah right,
Big organizations like VLC GitHub also do this same thing.
Move contributors to new read me and also long instructions or features.
Try your best to make read me as simple as possible.

As a visitor, when I visit read me file then I don't get what I am reading or what this page is for.
Try to make it clean and minimal.

**Pull request closed without merging**